dotty/matchers-core/src/main/scala/org/scalatest/matchers/dsl/NotWord.scala

```diff
@@ -54,7 +54,7 @@ final class NotWord {
    *                     ^
    * </pre>
    **/
-  def apply[S](matcher: Matcher[S]): Matcher[S] =
+  infix def apply[S](matcher: Matcher[S]): Matcher[S] =
     new Matcher[S] {
```

Comment:
This infix should be removed.

Reply:
@bvenners I have removed the related infix in the new commit that I just pushed.

Thanks for checking!